pkg/cmd/pr/view/view_test.go

```diff
@@ -18,12 +18,61 @@ import (
 	"github.com/cli/cli/v2/pkg/cmdutil"
 	"github.com/cli/cli/v2/pkg/httpmock"
 	"github.com/cli/cli/v2/pkg/iostreams"
+	"github.com/cli/cli/v2/pkg/jsonfieldstest"
 	"github.com/cli/cli/v2/test"
 	"github.com/google/shlex"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
+func TestJSONFields(t *testing.T) {
```

Comment:
❤️ 

If we ever add discussions, I imagine it would benefit from this work, too, as I think there is also technically some overlap of all 3 given their nature.